lambda/matching.ml

```diff
@@ -56,409 +58,509 @@ and may_compats = MayCompat.compats
      - Jump summaries: mapping from exit numbers to contexts
 *)
 
-
 let string_of_lam lam =
-  Printlambda.lambda Format.str_formatter lam ;
+  Printlambda.lambda Format.str_formatter lam;
   Format.flush_str_formatter ()
 
-let all_record_args lbls = match lbls with
-| (_,{lbl_all=lbl_all},_)::_ ->
-    let t =
-      Array.map
-        (fun lbl -> mknoloc (Longident.Lident "?temp?"), lbl,omega)
-        lbl_all in
-    List.iter
-      (fun ((_, lbl,_) as x) ->  t.(lbl.lbl_pos) <- x)
-      lbls ;
-    Array.to_list t
-|  _ -> fatal_error "Parmatch.all_record_args"
+let all_record_args lbls =
+  match lbls with
+  | (_, { lbl_all }, _) :: _ ->
+      let t =
+        Array.map
+          (fun lbl -> (mknoloc (Longident.Lident "?temp?"), lbl, omega))
+          lbl_all
+      in
+      List.iter (fun ((_, lbl, _) as x) -> t.(lbl.lbl_pos) <- x) lbls;
+      Array.to_list t
+  | _ -> fatal_error "Parmatch.all_record_args"
```

Comment:
`Matching.all_record_args` rather than `Parmatch. ... `

Reply:
Indeed. This has been here for two years now, but let's take the opportunity to change it.